### PR TITLE
fix(cd): Add new cd steps with buildx for lambda arm/amd images

### PR DESCRIPTION
**Pull request by adityasadalage**
# Problem #
Arm support needs lambda image to be updated for both architectures

# Solution #
Update Lambda cd steps to use buildx for both images to be tagged together

### CODE REVIEW

File: .github/workflows/cd-dgraph-lambda.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - uses: actions/checkout@v3
+      - name: Get Go Version
+        run: |
+          #!/bin/bash
+          GOVERSION=$({ [ -f .go-version ] && cat .go-version; })
+          echo "GOVERSION=$GOVERSION" >> $GITHUB_ENV
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GOVERSION }}
```
Comment (skrdgraph) [inline .github/workflows/cd-dgraph-lambda.yml:16]: remove go ... this is not needed

**Comment by adityasadalage** [comment]
> go can be removed

Done

File: .github/workflows/cd-dgraph-lambda.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD_TOKEN }}
+      - name: Push dgraph-lambda images
+        run: |
+          docker buildx create --name builder --driver docker-container
+          docker buildx use builder
+          docker buildx build -t dgraph/dgraph-lambda:1.4.0 --push --platform=linux/arm64,linux/amd64 .
```
Comment (skrdgraph) [inline .github/workflows/cd-dgraph-lambda.yml:31]: 1.4.0 -> change to env variable

**Review by skrdgraph — CHANGES_REQUESTED** [review]
fix version reference

**Review by skrdgraph — APPROVED** [review]
looks good 👍

**Review by joshua-goldstein — APPROVED** [review]
Looks good 👍

**Pull request merged**